arch/cortex-m0/src/lib.rs

```diff
@@ -114,6 +121,7 @@ core::arch::global_asm!(
     pop {{r4-r7}}
 
     ldr r0, 200f // MEXC_RETURN_MSP
+    mov lr, r0
```

Comment:
Can you comment on this additional change? The line before this was definitely pointless previously. But it's been like that for years apparently without an issue?

Reply:
I don't think it's necessarily been without issue, we just haven't had an armv6m platform since the nrf51 until the rp2040